extractor/extractor_options.cpp

```diff
@@ -64,7 +64,11 @@ ExtractorOptions::ParseArguments(int argc, char *argv[], ExtractorConfig &extrac
         "Number of threads to use")(
             "generate-edge-lookup",boost::program_options::value<bool>(
                                                 &extractor_config.generate_edge_lookup)->implicit_value(true)->default_value(false),
-                                 "Generate a lookup table for internal edge-expanded-edge IDs to OSM node pairs");
+                                 "Generate a lookup table for internal edge-expanded-edge IDs to OSM node pairs")(
+        "small-component-size",
+        boost::program_options::value<unsigned int>(&extractor_config.small_component_size)
+            ->default_value(1000),
+        "Number of nodes required before a strongly-connected-componennt is considered big (affects nearest neighbor snapping)");
```

Comment:
Cool.. but how does it affect the snapping? From the description I can not see how it would affect snapping if I increase the size, and how it affects the snapping if I decrease the size. I think we can be more user-friendly here --- in the end it's not just us who wants to use this.